eventuate-common-spring-reactive-jdbc/build.gradle

```diff
@@ -0,0 +1,20 @@
+
+dependencies {
+    compile project (":eventuate-common-spring-jdbc")
+    compile project (":eventuate-common-spring-id")
+
+    compile 'dev.miku:r2dbc-mysql:0.8.2.RELEASE'
```

Comment:
These versions need to be specified in gradle.properties.

Also, please provide links in the issue that explain/justify the choice of non-obvious dependencies, e.g. dev.miku:r2dbc-mysql